recipes/hypso/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "hypso" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NTNU-SmallSat-Lab/hypso-package/archive/v{{ version }}.tar.gz
+  sha256: 3be8366472536bc33a8a09d189a20b284124cc6660525f93129523ef83a50e0e
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+    - setuptools >=61.0
+    - setuptools_scm  >=3.4
+  run:
+    - python >=3.9
+    - numpy >=1.24
+    - gdal =3.6.2
+    - cartopy >=0.21.1
+    - pandas
+    - matplotlib
```

Unless you explicitly needs qt consider using just matplotlib-base instead of the heavier package with the qt bindings.

```suggestion
    - matplotlib-base
```

---

I did the modifications in the meta.yaml and the pyproject.toml (https://github.com/NTNU-SmallSat-Lab/hypso-package/blob/v1.0.1/pyproject.toml) but got this error from the test in the meta.yml file. Any ideas? 

```
import: 'hypso'
import: 'hypso'
+ pip check
hypso 1.0.1 requires matplotlib-base, which is not installed.
```

---

@DevAlvaroF there are some misconceptions here you should not change the name in the Python package metadata. There is no such thing as "matplotlib-base" in the Python world.

Conda is an agnostic package manager and, in the world of diverse packages, we can break them down to make them easier to install. In this case, the python package named matplotlib was broken down into the "full matplotlib", that few people really needs it, and the matplotlib-base that removes the qt dependency making it easier to install, lighter, and taking less space. B/c virtually everyone uses notebooks theses days, installing qt to have matplotlib is not desired. 

TL;DR you don't need a new release or any change upstream, just here you need to change matplotlib to matplotlib-base to avoid download qt b/c you probably don't need it.

---

Thanks for the explanation! I learn something new every time. All tests passed <3